examples/test_messaging/test_messaging.c

```diff
@@ -289,11 +290,12 @@ nf_msg_handler(void *msg_data, struct onvm_nf_local_ctx *nf_local_ctx) {
                                 printf("---------------------------\n");
                                 msg_params->tests_passed++;
                                 msg_params->test_phase++;
+                                msg_params->test_msg_count = 0;
                         }
                 }
         }
         if (4 == msg_params->test_phase) {
-                if ((int)rte_mempool_avail_count(msg_params->msg_pool) == msg_params->mempool_count) {
+                if ((int)rte_mempool_avail_count(msg_params->msg_pool) == msg_params->mempool_count - 1) {
                         printf("Passed %d/3 Tests\n", msg_params->tests_passed);
```

Comment:
need to update all of these to be `Passed %d/4` instead of `/3` since you added another test.